NOTICE

```diff
@@ -118,3 +118,14 @@ taverna-ui/src/main/java/org/apache/taverna/lang/ui/treetable/
   for use in the design, construction, operation or maintenance of any
   nuclear facility.
 
+---------------------------------------------------------
+incubator-taverna-workbench/taverna-perspective-biocatalogue/src/main/resources/net/sf/taverna/t2/ui/perspectives/biocatalogue/service_icons/soap_rest_multitype_unknown.pdn
+
+TODO: License should be clarified
+
+---------------------------------------------------------
+incubator-taverna-workbench/taverna-perspective-biocatalogue/src/main/resources/net/sf/taverna/t2/ui/perspectives/biocatalogue/famfamfam_silk/add - tick.pdn
```

Comment:
How can we find, there is no document mentioning under which license those Icons are ?